### PR TITLE
ATLAS-139: Edit bootstrap.sql with types and versions database schema

### PR DESCRIPTION
**Pull request by HelioStrike**
Created tables for types and versions using bootstrap.sql .

### CODE REVIEW

**Comment by HelioStrike** [comment]
@cintiadr A new file containing only types and versions right?

**Comment by cintiadr** [comment]
Yeah, that

**Comment by HelioStrike** [comment]
@cintiadr How about now? :)

File: db/bootstrap.v3.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+--
+-- Table structure for table `types`
+--
+
+DROP TABLE IF EXISTS `types`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `types` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(50) NOT NULL,
```
Comment (bmamlin) [inline db/bootstrap.v3.sql]: Assuming this is the same as `atlas.type`, I would suggest `varchar(1024)` for consistency.

File: db/bootstrap.v3.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  `icon` varchar(400) NOT NULL,
```
Comment (bmamlin) [inline db/bootstrap.v3.sql]: Assuming this is a URL, I would suggest `varchar(1024)` for consistency.[[ref](https://github.com/HelioStrike/openmrs-contrib-atlas/blob/ATLAS-139/db/bootstrap.sql#L81)]

File: db/bootstrap.v3.sql [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```
Comment (bmamlin) [inline db/bootstrap.v3.sql]: Use UTF-8 like [this](https://github.com/HelioStrike/openmrs-contrib-atlas/blob/ATLAS-139/db/bootstrap.sql#L101).

File: db/bootstrap.v3.sql [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+--
+
+DROP TABLE IF EXISTS `versions`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `versions` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `version` varchar(50) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```
Comment (bmamlin) [inline db/bootstrap.v3.sql]: Ditto... UTF-8.

**Comment by HelioStrike** [comment]
@bmamlin I made the changes :)

**Pull request merged**